tests/slow/src/test/scala/tests/sbt/SbtServerSuite.scala

```diff
@@ -32,10 +33,13 @@ class SbtServerSuite
   val scalaVersion = V.scala213
   val buildTool: SbtBuildTool = SbtBuildTool(None, () => userConfig)
   var compilationCount = 0
+  val bspStartLogs: ListBuffer[String] = ListBuffer.empty[String]
+  override val handleBspStartupErrorMessage = bspStartLogs.append
```

Comment:
Can you use `workspaceLogMessages` instead? I would avoid passing around the function if possible